WalletWasabi/WabiSabi/Backend/Events/InputAddedEventArgs.cs

```diff
@@ -0,0 +1,19 @@
+using NBitcoin;
+using WalletWasabi.Affiliation;
+using WalletWasabi.WabiSabi.Backend.Rounds;
+
+namespace WalletWasabi.WabiSabi.Backend.Events;
+
+public class InputAddedEventArgs : EventArgs
+{
+	public InputAddedEventArgs(uint256 roundId, Coin coin, bool isCoordinationFeeExempted) : base()
```

Comment:
nop, i think it is important to be clear about what fee we are talking about, that's why in all the code we have miningFee and coordinationFee instead of simply fee.